content/03.methods.md

```diff
@@ -14,9 +14,35 @@ talk about generative model and how it works
 ### Word Embeddings
 mention facebooks fasttext model and how we used it to train word vectors
 ### Discriminator Model
-talk about the discriminator model and how it works
+The discriminator model is designed to improve off of the predctions from the generative model.
```

Comment:
"predctions" misspelling. Is it confusing to call the generative output "predictions"? Should we reserve the word predictions for the discriminator output? Let's try to find a consistent terminology for the output value from the generative model.

Reply:
How about "generated labels"?